### PR TITLE
Quick scripts

### PR DESCRIPTION
**Pull request by iturricf**
add quick scripts to optimize workflow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
     "build": "tsc --build tsconfig.json && yarn workspaces foreach -A run build",
+    "build:sdk-evm": "yarn sdk/evm build",
+    "build:sdk-solana": "yarn sdk/solana build",
+    "build:solana-devenet": "yarn sdk/solana build:devnet",
+    "build:solana": "yarn sdk/solana build",
+    "build:evm": "(cd evm && forge build)",
+    "build:all-devnet": "yarn build:solana-devenet && yarn build:evm && yarn build:sdk-solana && yarn build:sdk-evm",
+    "build:all": "yarn build:solana && yarn build:evm && yarn build:sdk-solana && yarn build:sdk-evm",
+    "clean:solana": "yarn sdk/solana clean",
+    "clean:evm": "(cd evm && forge clean)",
```
Comment (solanoepalacio) [inline package.json:16]: we need to remove the typechain bindings as well... both for `/evm` and `/deployment` (I'm not sure if the latter is using the sdk ones or generating one of their own)

File: package.json [hunk 2]
```diff
@@ -5,6 +5,15 @@
   "description": "",
   "scripts": {
     "build": "tsc --build tsconfig.json && yarn workspaces foreach -A run build",
+    "build:sdk-evm": "yarn sdk/evm build",
+    "build:sdk-solana": "yarn sdk/solana build",
+    "build:solana-devenet": "yarn sdk/solana build:devnet",
```
Comment (solanoepalacio) [inline package.json:10]: there's a typo here

File: solana/package.json [hunk 3]
```diff
@@ -2,7 +2,9 @@
   "scripts": {
     "lint:fix": "prettier */*.js \"*/**/*{.js,.ts}\" -w",
     "lint": "prettier */*.js \"*/**/*{.js,.ts}\" --check",
-    "build": "yarn ../sdk/solana/ build && tsc --project tsconfig.json",
+    "build:devnet": "anchor build -- --features solana-devnet && rm ./sdk/solana/tbrv3/idl/token_bridge_relayer.json && cp ./target/idl/token_bridge_relayer.json ./sdk/solana/tbrv3/idl",
+    "build": "anchor build && rm ./sdk/solana/tbrv3/idl/token_bridge_relayer.json && cp ./target/idl/token_bridge_relayer.json ./sdk/solana/tbrv3/idl",
+    "clean": "rm -rf target/idl target/types",
```
Comment (solanoepalacio) [inline solana/package.json:7]: I think the `target` directory is created at the root of the repo

File: package.json (hunk 1 above)
Comment (iturricf) [reply package.json:17]: only the deployment is generating and using the typechain. I created scripts for that

File: package.json (hunk 2 above)
Comment (iturricf) [reply package.json:10]: thanks. fixed

File: package.json [hunk 4]
```diff
@@ -5,6 +5,15 @@
...
+    "build:solana": "yarn sdk/solana build",
+    "build:evm": "(cd evm && forge build)",
```
[the remark this answers is not in the document]
Comment (iturricf) [reply package.json:12]: now it builds the typechain along with the evm contracts

File: solana/package.json (hunk 3 above)
Comment (iturricf) [reply solana/package.json:7]: yes, fixed it

**Review by brianUtn98 — APPROVED** [review]

**Pull request merged**